docs/release/3.5/release-3-5-14.md

```diff
@@ -94,3 +94,34 @@ minicondor-8.8.8-1.osg35.el7
 python2-condor-8.8.8-1.osg35.el7
 python3-condor-8.8.8-1.osg35.el7
 ```
+
+### Upcoming Packages
+
+We added or updated the following packages to the **upcoming** OSG yum repository. Note that in some cases, there are multiple RPMs for each package. You can click on any given package to see the set of RPMs or see the complete list below.
+
+-   [condor-8.9.6-1.osgup.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=condor-8.9.6-1.osgup.el7)
+
+### Upcoming RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    blahp blahp-debuginfo condor condor-all condor-annex-ec2 condor-bosco condor-classads condor-classads-devel condor-debuginfo condor-kbdd condor-procd condor-test condor-vm-gahp GenericError: Invalid tagInfo:
```

Comment:
Why does list include the blahp? And the `GenericError` business at the end?

Reply:
The script is buggy when it can find the previous version. It sometimes includes more than it should. The Generic error is due to the koji update when the script can find the previous version. I have updated the script in the release tools.